api/src/main/java/org/openmrs/module/coreapps/utils/VisitTypeHelper.java

```diff
@@ -189,76 +181,4 @@ public Map<Integer, Map<String, Object>> getVisitColorAndShortName(List<VisitDom
 		}
 		return visitsWithAttr;
 	}
-
-	/**
-	 * Sets encounters based on visit type and login location
```

Comment:
These are the ones that aren't used?   Looks like they were added for this: https://openmrs.atlassian.net/browse/RA-977, but that seems to all be tied to stuff in this module (I was trying to check to see if it was used by a dependent module).
tl;dr... yeah, if these aren't used within coreapps, good to remove

Reply:
Yes.  Basically, these methods are trying to create an encounter, and to set a provider on an encounter, and this is no longer possible in 2.x because we need to know what encounter role to use when assigning the provider.  But then I realized that these methods were not actually used, so I could remove them and not have to figure out how to overcome this incompatibility, so I did.  If something downstream was using them, we can adapt there I hope.